src/dom.js

```diff
@@ -926,6 +926,21 @@ export function toggleAttribute(element, name, forced) {
   return enabled;
 }
 
+/**
+ * Parses a string as a boolean value using the expanded rules for DOM boolean
+ * attributes:
+ * - a `null` or `undefined` returns `null`;
+ * - an empty string returns `true`;
+ * - a "true" string returns `true`;
+ * - otherwise, `false` is returned.
+ *
+ * @param {?string|undefined} s
+ * @return {?boolean}
+ */
+export function parseBooleanAttribute(s) {
+  return s == null ? null : s === '' || s === 'true';
```

Comment:
`checked=checked` is also common way to enable. I think we actually have this backwards, only explicit `false` should disable (and absent -> `undefined`).